testsuite/features/step_definitions/common_steps.rb

```diff
@@ -371,22 +371,69 @@
 
 # setup wizard
 
-When(/^I make the credentials primary$/) do
-  raise 'xpath: i.fa-star-o not found' unless find('i.fa-star-o').click
+Then(/^HTTP proxy verification should have succeeded$/) do
+  raise 'Success icon not found' unless find('i.text-success', wait: DEFAULT_TIMEOUT)
```

Comment:
I still confused every time I see DEFAULT_TIMEOUT specified somewhere xD
But, yes, it is different from `Capybara.default_max_wait_time = 10``:)

Reply:
It has nothing to do with capybara, it is the default "long" timeout (4 minutes or such).